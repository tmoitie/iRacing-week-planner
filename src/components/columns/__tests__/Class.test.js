import React from 'react';
import { render, screen } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import { TableWrapper } from './ColumnUtils';

import Class from '../Class';

describe('components/columns/Class', () => {
  test('renders correctly', () => {
    render(<TableWrapper><Class race={{ licenceLevel: 5 }} /></TableWrapper>);

    expect(screen.getByTestId('table-row').firstChild).toMatchSnapshot();
  });
});
