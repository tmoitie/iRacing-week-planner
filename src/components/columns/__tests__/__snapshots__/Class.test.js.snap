// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Class renders correctly 1`] = `
<td>
  <div
    class="licenceLevelComponent licenceD"
  >
    <span
      class="licenceLetter"
    >
      D
    </span>
    <span
      class="licenceText"
    />
  </div>
</td>
`;
